docs/reference/settings.md

```diff
@@ -412,14 +412,47 @@ Custom storage classes should subclass `django.core.files.storage.Storage`. See
 ### `WAGTAILIMAGES_EXTENSIONS`
 
 ```python
-WAGTAILIMAGES_EXTENSIONS = ['png', 'jpg']
+WAGTAILIMAGES_EXTENSIONS = ['avif', 'svg']
```

Comment:
Not precious about which extensions we have in there, was just trying to go some way towards avoiding #13127